src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreTraceInstrumentationOptions.cs

```diff
@@ -29,6 +29,15 @@ internal AspNetCoreTraceInstrumentationOptions(IConfiguration configuration)
         {
             this.EnableGrpcAspNetCoreSupport = enableGrpcInstrumentation;
         }
+
+        if (configuration.TryGetStringValue("OTEL_INSTRUMENTATION_HTTP_KNOWN_METHODS", out var knownHttpMethods))
+        {
+            this.KnownHttpMethods = knownHttpMethods
+                .Split(',')
+                .Select(x => x.Trim())
+                .Where(x => !string.IsNullOrEmpty(x))
+                .ToList();
```

Comment:
I don't _love_ that this code is also duplicated in the ctor on `RequestMethodHelper`. Why not just call that ctor here to avoid the duplication?

```csharp
this.KnownHttpMethods = new RequestMethodHelper(knownHttpMethods);
```